ledfx/utils.py

```diff
@@ -26,6 +27,29 @@
 
 _LOGGER = logging.getLogger(__name__)
 
+sleep_lies = True
+
+
+def does_sleep_lie():
+    global sleep_lies
+    monotonic_res = time.get_clock_info("monotonic").resolution
+    perf_counter = time.get_clock_info("perf_counter").resolution
+    _LOGGER.warning(f"mono: {monotonic_res} perf: {perf_counter}")
+    count = 0
+    start = timeit.default_timer()
+    time.sleep(0.001)
+    sleep_time = timeit.default_timer() - start
+    _LOGGER.warning(f"{count} sleep_time for 0.001: {sleep_time}")
+
+    if sleep_time < (monotonic_res / 2):
```

Comment:
This test means we only get new behavior if the measured sleep time is less that half the reported monotonic time. There fore all the systems with good timings will be unaffected and / or very short monotonics will be unaffected.